src/igvfd/searches/configs/tissue.py

```diff
@@ -99,9 +187,8 @@ def tissue():
                     'audit.WARNING.category',
                     'audit.INTERNAL_ACTION.category',
                 ]
-            },
-        ],
-        'columns': {
+            }
+        ], 'columns': {
```

Comment:
I think this might be an accidental change? It looked OK with the extra line break previously.